sitemap.xml

```diff
@@ -41,7 +41,7 @@
 	<priority>0.8</priority>
 </url>
 <url>
-	<loc>https://purdueieee.org/part/</loc>
+	<loc>https://purdueieee.org/neurotech/</loc>

```

Comment:
Don't modify the sitemap like this, [use a site like this](https://www.xml-sitemaps.com/) instead

Reply:
We'll do this after we get this PR pushed